### PR TITLE
Add contact form and finish mobile version

### PR DESCRIPTION
**Pull request by JonahKayizzi**
This pull request includes the following changes.
- Added contact section to the portfolio mobile version
- Added a contact form with form validation
- Implemented form submission using [Formspree service](https://formspree.io/html)
- Demonstrated pair programming by making changes to the repository together with a coding partner.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -155,5 +155,28 @@ <h4 class="about-card-title">Skills</h4>
...
         </div>
     </section>
+    <section class="contact-section">
+        <div class="contact-heading">
+            <h2 class="home-headings contact-title">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        </div>
+        <form action="https://formspree.io/f/xvoygkpa" class="contact-form" method="post">
+            <input type="text" name="name" id="name" maxlength="30" placeholder="Full name" required>
+            <input type="email" name="email" id="email" placeholder="Email address" required>
+            <textarea name="message" id="message" cols="30" rows="4" placeholder="Enter text here" required></textarea>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [ ] Kindly, try to update Message input to have `maxlength="500"` attribute.

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @JonahKayizzi   , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional README
✔️ Good commit messages

almost

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (JonahKayizzi) [reply index.html]: Oh yes, that you very much for this reminder 😄

**Review by BertrandConxy — APPROVED** [review]
Hi @JonahKayizzi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 
- No linter errors
- project requirements are met
- project works fine
- good PR description
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**